### ISSUE TITLE
Aria-required property is missing for required field in Home screen.

### ISSUE THREAD

**Issue by rgarg24** [percussion/percussioncms#876]
**Describe the bug**
Aria-required property is missing for required field.

**To Reproduce**
1. Go to Home screen (http://localhost:9992/cm/app/?view=home&pathType=unknown)
2. Click on "Add New" button. Then in the "Add Content" popup, select "Blog Post" and then click on "Next" button.
3. "Blog Post Title" is a required field. Inspect the "Blog Post Title" text box element, you will find that the Aria-required=true property is missing for the input text.

**Expected behavior**
Aria-required=true property must be available for all required fields.

**Additional context**
Navigate all the options available for similar issue and fix.
